src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -36,15 +37,27 @@ public class Elevator extends SubsystemBase implements DefaultInternallyControll
 
     private Elevator() {
       master = new CANSparkMax(PortMap.Elevator.masterID, MotorType.kBrushless);
-      slave = new CANSparkMax(PortMap.Elevator.slaveID, MotorType.kBrushless);
+      slave1 = new CANSparkMax(PortMap.Elevator.slave1ID, MotorType.kBrushless);
+      slave2 = new CANSparkMax(PortMap.Elevator.slave2ID, MotorType.kBrushless);
+      slave1.restoreFactoryDefaults();
+      slave2.restoreFactoryDefaults();
+      master.restoreFactoryDefaults();
+
+      absEncoder = new AnalogEncoder(PortMap.Elevator.absEncoderID);
 
-      master.setIdleMode(IdleMode.kBrake);
-      slave.setIdleMode(IdleMode.kBrake);
+      master.setIdleMode(IdleMode.kCoast);
+      slave1.setIdleMode(IdleMode.kCoast);
+      slave2.setIdleMode(IdleMode.kCoast);
+      slave1.follow(master, true);
+      slave2.follow(master, true);
 
       encoder = master.getEncoder();
 
       encoder.setPositionConversionFactor(ElevatorConstants.positionConversionFactor);
 
+      resetPose(absEncoder.getAbsolutePosition() * 
+        ElevatorConstants.absPositionConversionFactor - ElevatorConstants.encoderOffset);
+
       resetPose(0);
```

Comment:
You do this calculation but then you immediately reset the pose to **0** 

```suggestion
      resetPose(absEncoder.getAbsolutePosition() * 
        ElevatorConstants.absPositionConversionFactor - ElevatorConstants.encoderOffset);
```